python/qpid_dispatch_internal/management/config.py

```diff
@@ -67,6 +67,7 @@ def transform_sections(sections):
             if s[0] == "autoLink":  s[0] = "router.config.autoLink"
             if s[0] == "exchange":  s[0] = "router.config.exchange"
             if s[0] == "binding":   s[0] = "router.config.binding"
+            if s[0] == "attachSubscription":   s[0] = "router.connection.attachSubscription"
```

Comment:
Why go to all the trouble of adding support for configuration of a non-configurable entity?

Reply:
See previous.